* plugins/media-keys/Makefile.am: Only install the .g-s-plugin file if
the plugin is actually installed.

--- origsrc/gnome-settings-daemon-2.26.0/plugins/media-keys/Makefile.am	2009-03-16 09:35:38.000000000 -0500
+++ src/gnome-settings-daemon-2.26.0/plugins/media-keys/Makefile.am	2009-04-03 02:36:09.743750000 -0500
@@ -62,7 +62,9 @@ libmedia_keys_la_LIBADD  = 		\
 plugin_in_files = 		\
 	media-keys.gnome-settings-plugin.in
 
+if HAVE_PULSE
 plugin_DATA = $(plugin_in_files:.gnome-settings-plugin.in=.gnome-settings-plugin)
+endif
 
 noinst_PROGRAMS =				\
 	test-media-keys				\
